rotkehlchen/tests/unit/test_loopring.py

```diff
@@ -43,17 +43,23 @@ def test_loopring_accountid_mapping(database):
     addr1 = make_evm_address()
     addr2 = make_evm_address()
 
-    with database.user_write() as cursor:
+    with database.conn.read_ctx() as cursor:
```

Comment:
isn't it better in cases like this test to open the read/write cursor that you have?

Reply:
reader can be used like that, but writer has to commit changes